Adyen/UI/Form/Items/Address/FormAddressItemView.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import UIKit
+
+/// A view representing a address item.
+internal final class FormAddressItemView: FormItemView<AnyAddressItem> {
```

Comment:
this is just a  vertical stack item, right?

maybe rebrand it as `VStackItemView`.

Reply:
I am still note sure if I need to do something extra here 🤔
But sure - that is a good idea!

Reply:
you didn't resolve this one, just rename it to `FormVerticalStackItemView ` and it takes any `FormItem`:

```
internal final class FormVerticalStackItemView: FormItemView<FormItem>
```

Reply:
👍